template/52497

```diff
@@ -0,0 +1,27 @@
+Welcome to the W3C {{ group.name }}, known as PING.
+
+We are delighted to have you join us. Our mission is to improve privacy on the Web.
+
+Our co-chairs are Pete Snyder (Brave) and Christine Runnegar (invited expert). Sam Weiler is our W3C contact.
+
+A couple of items of information that may be helpful.
+
+* You have been subscribed to the public W3C privacy activity email list, used by PING, public-privacy@w3.org. We will invite you to the PING Slack workspace. Please look for a separate email.
+
+* We meet the 1st and 3rd Thursday of each month.
```

Comment:
This should be "We meet the 1st and 3rd Thursday of each month at UTC 17.

Reply:
Except for DST...  I'll edit...